### ISSUE TITLE
Today button does nothing

### ISSUE THREAD

**Issue by bjm904** [mui-org/material-ui-pickers#331]
## Environment

| Tech                | Version |
|---------------------|---------|
| material-ui-pickers | material-ui-pickers@1.0.0-rc.5 |
| material-ui         | material-ui@1.0.0-beta.40 |
| React               | 16.3.1|
| Browser             | Chrome 65.0.3325.181 |
| Platform            | Windows 10.0.16299 |

## Steps to reproduce
1. Create DateTimePicker with showTodayButton prop

## Expected behavior
Go to today's date

## Actual behavior
Nothing happens. No errors in log either

I am using 

```
import { DateTimePicker } from 'material-ui-pickers';
import MomentUtils from 'material-ui-pickers/utils/moment-utils';
import MuiPickersUtilsProvider from 'material-ui-pickers/utils/MuiPickersUtilsProvider';
import IconAccessTime from 'material-ui-icons/AccessTime';
import IconDateRange from 'material-ui-icons/DateRange';
import IconKeyboardArrowLeft from 'material-ui-icons/KeyboardArrowLeft';
import IconKeyboardArrowRight from 'material-ui-icons/KeyboardArrowRight';

<MuiPickersUtilsProvider utils={MomentUtils}>
<DateTimePicker
                autoOk
                dateRangeIcon={<IconDateRange />}
                label="Date Time"
                leftArrowIcon={<IconKeyboardArrowLeft />}
                onChange={this.handleDateChange}
                rightArrowIcon={<IconKeyboardArrowRight />}
                showTodayButton
                timeIcon={<IconAccessTime />}
                value={new Date(this.state.date)}
/>
</MuiPickersUtilsProvider>
```

**Comment by bjm904** [comment]
Note: The 'MuiPickersUtilsProvider' is in another file at root of app, I just included it here for completeness

**Comment by Loktor** [comment]
I have a similar problem, but for me the today button actually selects the current date (but doesn't close the dialog), which might be a bit of an confusing behaviour. Also it selects the value e.q. if you abort the dialog the click on the okay button still selected the value of today e.q. abort is in that case useless

**Comment by Loktor** [comment]
@hamzahayat maybe to clarify it better, i would have expected that the today button behaves like the other day buttons of the calendar (e.q. they seem to select the value internally but don't set the value for the datePicker control yet, only after the user klicks the okay button, e.q. if the user clicks today + then abort then the today value gets selected even if the user aborts the dialog afterwards (i think this is not intended behaviour or? But an option to close the dialog with the today button would be nice too (maybe close it automatically if the user has set autoOk to true?)

As for the second issue i think this one is not necessary a bug but, it would be nice to have a feature where you can have the today button + be able to clear the control via input (if you have selected the keyboard option)

I probably should make separate tickets for these things^^
